src/aria/combobox/combobox.ts

```diff
@@ -113,6 +113,21 @@ export class Combobox<V> {
       }
     });
   }
+
+  /** Opens the combobox. */
+  open(nav?: {first?: boolean; last?: boolean; selected?: boolean}) {
```

Comment:
We should discuss with the team how these navigation options are surfaced to developers. When it was an internal impl detail, it was fine to just call `open({first: true})`, etc. But now that it is part of the public api, it would be good to see if folks think there is a more intuitive way to expose this

Reply:
That's fair. I will add that the context-menu example appears to call _pattern.first() because there was no option to open and navigate to first, so at least providing it in the open (or simply making the default first: true) will remove the need to expose navigation separately. But again, just having it clearly navigate to first would possibly be enough.

Reply:
Agreed to remove options and go with default behavior.